src/tests/checkers/catch_test.go

```diff
@@ -110,3 +110,111 @@ try {
 }
 `)
 }
+
+func TestTryCatchVariables(t *testing.T) {
+	test := linttest.NewSuite(t)
+	test.AddFile(`<?php
+function f() {
+	try {
+		$a = 100;
+		$c = 100;
+		$d = 100;
+	} catch (Exception $_) {
+		$a = 200;
+	} catch (ExceptionDerived $_) {
```

Comment:
It seems like we don't give "undefined class" warnings for catch clauses?
This test case should probably define `ExceptionDerived` or do a RunFilterMatch so it's less prone to unrelated changes.

Reply:
> It seems like we don't give "undefined class" warnings for catch clauses?

It seems so.